relay-common/src/metrics.rs

```diff
@@ -88,8 +128,11 @@ pub mod prelude {
 }
 
 /// Set a new statsd client.
-pub fn set_client(statsd_client: StatsdClient) {
-    *METRICS_CLIENT.write() = Some(Arc::new(statsd_client));
+pub fn set_client(statsd_client: StatsdClient, default_tags: BTreeMap<String, String>) {
```

Comment:
_nit_: Since `MetricsClient` is public, you can just pass that in here and construct outside of the function. That will keep the signature stable going forward.